src/page-module/files/__name@dasherize__/__name@dasherize__.component.ts

```diff
@@ -7,5 +9,11 @@ import { Component<% if (isNgrxInstalled) { %>, ChangeDetectionStrategy<% } %> }
   changeDetection: ChangeDetectionStrategy.OnPush<% } %>
 })
 export class <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageComponent {
+<% if (facade) { %>  public isLoading$: Observable<boolean>;
 
+  constructor(
+    private facade: <%= classify(section + ' ' + ((hasParent) ? (parent + ' ') : '') + name) %>PageFacade
```

Comment:
@Goodmain same as above, this `classify` call can be shortened.

Reply:
@dmitryusenko Fixed